src/cursor/cursor.ts

```diff
@@ -264,7 +492,207 @@ export class Cursor<
       }
     }
 
-    super._initializeCursor(callback);
+    this._initializeCoreCursor(callback);
+  }
+
+  /** @internal */
+  _initializeCoreCursor(callback: Callback): void {
```

Comment:
I'm fine leaving `_initializeCursor` and `_initializeCoreCursor` split for just this PR, but hope that the plan is to merge the two?

Reply:
This was an oversight on my part, they are now merged.